### ISSUE TITLE
Install issue....

### ISSUE THREAD

**Issue by aelkman** [sammchardy/python-binance#46]
```
3.4\twisted\words\im
    copying src\twisted\words\xish\xpathparser.g -> build\lib.win-amd64-3.4\twis
ted\words\xish
    running build_ext
    building 'twisted.test.raiser' extension
    error: Microsoft Visual C++ 10.0 is required. Get it with "Microsoft Windows
 SDK 7.1": www.microsoft.com/download/details.aspx?id=8279

    ----------------------------------------
Command "c:\python34\python.exe -u -c "import setuptools, tokenize;__file__='C:\
\Users\\Alex\\AppData\\Local\\Temp\\pip-build-3scyxu0w\\Twisted\\setup.py';f=get
attr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.clo
se();exec(compile(code, __file__, 'exec'))" install --record C:\Users\Alex\AppDa
ta\Local\Temp\pip-aeidjyp5-record\install-record.txt --single-version-externally
-managed --compile" failed with error code 1 in C:\Users\Alex\AppData\Local\Temp
\pip-build-3scyxu0w\Twisted\
```

I have tried setting my location of my env var as described here, still doesn't fix the issue.
[Proposed solutions here](https://stackoverflow.com/questions/28251314/error-microsoft-visual-c-10-0-is-required-unable-to-find-vcvarsall-bat)

**Comment by aelkman** [comment]
Resolved issue by upgrading Python to 3.6 and installing the .NET 2015 Dev pack

**Comment by Hujjat** [comment]
Install Visual Studio 2015, it will solve the problem. As the error says, go to that link 
www.microsoft.com/download/details.aspx?id=8279
